packages/cli/tailwind.config.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
         "gray-500": "#555",
         "gray-600": "#444",
         "gray-700": "#333",
+        mint: "#C3F2BC",
```

Comment:
@psugihara is this naming ok these 3 lines?

Reply:
I think let's try to match tailwind color names. cc @monicakogler who did those and is trying to keep them in sync w Figma colors.

Reply:
tia @monicakogler , these are the button colors on the Account page

Reply:
cool, assuming this is for the button. the text is `green-300` and the border is `emerald-700`. hopefully it'll be easier now to know the colors from figma

Reply:
Thanks